README.rst

```diff
@@ -74,7 +74,7 @@ The full documentation is hosted by readthedoc, here : http://patacrep.readthedo
 Contact & Forums
 ----------------
 
-* http://www.patacrep.com/forum
+* http://www.patacrep.fr/forum
```

Comment:
URL seems dead, might as well remove it?

Reply:
possibly yes. Or shall we fix the forum ? (or remove forum from website). 
Who manages website ?

Reply:
@crep4ever is the owner of the website.

The forum link can probably be replaced with a link to github issues.